### PR TITLE
Ff112 remove mutable db

### PR DESCRIPTION
**Pull request by hamishwillee**
FF112 permanently removes `IDBMutableFile`, `IDBFileRequest`, `IDBFileHandle`, `IDBDatabase.createMutableFile()` from FF in https://bugzilla.mozilla.org/show_bug.cgi?id=1500343

These had been behind a pref in 102 and had been seeing very little use for some years prior. They also are not used in other browsers for similar timeframes. There is no BCD. They are also went through several renames and have been confusing to users. 
For these reasons, I have removed the entries as suggested by @teoli2003 here https://github.com/mdn/content/issues/25360#issuecomment-1471376700 rather than waiting the customary 2 years (i.e. we _know_ no one is relying on these, so there is no real value in keeping them).

This removes all the entries for `IDBMutableFile`, `IDBFileRequest`, `IDBFileHandle`, `IDBDatabase.createMutableFile()`. It also removes all mention in normal text. In release notes it keeps the mentions, but removes any links. 
The redirect is to the top level http://localhost:5042/en-US/docs/Web/API/IndexedDB_API page.

Related docs work can be tracked in #25360

### CODE REVIEW

**Review by rebloor — APPROVED** [review]
Looks good for the webextension changes.

**Comment by hamishwillee** [comment]
@wbamberg Do you think you could have a quick check of this.

File: files/en-us/mozilla/firefox/releases/102/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ No notable changes.
 
 ### APIs
 
-- The Non-standard interfaces [IDBMutableFile](/en-US/docs/Web/API/IDBMutableFile), [IDBFileHandle](/en-US/docs/Web/API/IDBFileHandle), [IDBFileRequest](/en-US/docs/Web/API/IDBFileRequest), and the method [IDBDatabase.createMutableFile()](/en-US/docs/Web/API/IDBDatabase#idbdatabase.createmutablefile) have been disabled by default in preparation for removal in a future release ([Firefox bug 1764771](https://bugzil.la/1764771)).
+- The Non-standard interfaces `IDBMutableFile`, `IDBFileHandle`, `IDBFileRequest`, and the method `IDBDatabase.createMutableFile()` have been disabled by default in preparation for removal in a future release ([Firefox bug 1764771](https://bugzil.la/1764771)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/102/index.md:26]: Nit
```suggestion
- The non-standard interfaces `IDBMutableFile`, `IDBFileHandle`, `IDBFileRequest`, and the method `IDBDatabase.createMutableFile()` have been disabled by default in preparation for removal in a future release ([Firefox bug 1764771](https://bugzil.la/1764771)).
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A nit, and this is good for me.

(I'm happy to see this go, I spent some time last year to untangle what happened here over the years)

**Comment by hamishwillee** [comment]
Awesome. A happy day then @teoli2003 !

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**